draft-ietf-tls-tls13.md

```diff
@@ -1370,8 +1369,8 @@ For example:
        struct {
            VariantTag type;
            select (VariantRecord.type) {
-               case apple:  V1;  /* VariantBody, tag = apple */
-               case orange: V2;  /* VariantBody, tag = orange */
+               case apple:  V1;
+               case orange: V2;
            } variant_body;       /* optional label on variant */
```

Comment:
The tag of `select` is only used in `Handshake` where the `body` tag is not referred at all.
We could remove the `body` tag of `Handshake`.